src/apify_client/clients/resource_clients/actor_collection.py

```diff
@@ -1,6 +1,8 @@
 from typing import Any, Dict, List, Optional
 
-from ..._utils import ListPage, _filter_out_none_values_recursively, ignore_docs
+from apify_shared.utils import filter_out_none_values_recursively, ignore_docs
+
+from ..._utils import ListPage
```

Comment:
We decided to keep it in the client because I thought it's kinda client-specific, it's not any shared logic.

But now looking at why we use it in the SDK, that was a mistake, we need it for the local storage emulation, so it's not really client-specific anymore, so we should put it in the shared package.

Question is, where? It's not really a utility, maybe we should have some `apify_shared.models`?

Reply:
Ok, I'm adding `models.ListPage` to `apify-shared` in https://github.com/apify/apify-shared-python/pull/4.